### DISCUSSION TITLE
Exodus Bitcoin Wallet: $490K Swindle - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#49] [General]
# Exodus Bitcoin Wallet: $490K Swindle - Alan Pope's blog

Edit: There’s a short follow-up to this post: Exodus Bitcoin Wallet: Follow up.
tl;dr: A Bitcoin investor was recently scammed out of 9 Bitcoin (worth around $490K) in a fake “Exodus wallet” desktop application for Linux, published in the Canonical Snap Store. This isn’t the first time, and if nothing changes, it likely won’t be the last.
This post turned out longer than I expected. So if you don’t have the time there’s a briefer summary at the bottom under “In summary (the tl;dr)” along with my suggestions on what Canonical should do now.

https://popey.com/blog/2024/02/exodus-bitcoin-wallet-490k-swindle/

**Comment by faveoled** [comment]
Should probably make all apps look "insecure" on the app store if their domain name is not confirmed (and checked by humans)

**Comment by ToS0** [comment]
Thank you - looks like a mistake from Canonical to let you go

**Comment by tcreek** [comment]
It appears Canonical owes this person $500k.
